modules/browsemap/src/main/java/com/sygic/modules/browsemap/viewmodel/BrowseMapFragmentViewModel.kt

```diff
@@ -88,6 +95,9 @@ class BrowseMapFragmentViewModel internal constructor(
     }
 
     override fun onMapObjectsReceived(viewObjects: List<ViewObject>) {
+        poiDetailsView?.let { extendedMapDataModel.removeMapObject(it) }
```

Comment:
Nechtěli bychom tedy rovnou aplikovat to o čem jsme se bavili, že pokud je poiDetailsView viditelné (!= null), tak by klik rovnou oba schoval a returnul? bude se ale muset checknout ještě i zda není nastavený onMapClickListener.

Reply:
myslim ze nie je nutne ani checkovat click listener kedze ten ak je nastaveny, k poiDetailsView sa to nedostane, cize by sa to nemalo bit